base_tier_validation/models/tier_validation.py

```diff
@@ -471,15 +479,9 @@ def request_validation(self):
                     for td in tier_definitions:
                         if rec.evaluate_tier(td):
                             sequence += 1
-                            created_trs += tr_obj.create(
-                                {
-                                    "model": self._name,
-                                    "res_id": rec.id,
-                                    "definition_id": td.id,
-                                    "sequence": sequence,
-                                    "requested_by": self.env.uid,
-                                }
-                            )
+                            vals = rec._prepare_tier_review_vals(td)
+                            vals.update(sequence=sequence)
```

Comment:
I think the sequence should be passed by argument to the prepare method, as you may want to customize it in any case.